ServerCore/Startup.cs

```diff
@@ -34,9 +36,21 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddAuthentication().AddMicrosoftAccount(microsoftOptions =>
             {
-                microsoftOptions.ClientId = Configuration["Authentication:Microsoft:ApplicationId"];
-                microsoftOptions.ClientSecret = Configuration["Authentication:Microsoft:Password"];
+                microsoftOptions.ClientId = Configuration["Authentication-Microsoft-ApplicationId"];
```

Comment:
Make sure to update the wiki and tell everyone about this change

Reply:
Wiki updated, email will go out in a few minutes & I'll tell people tonight.